src/python/grpcio/README.rst

```diff
@@ -60,7 +60,7 @@ package named :code:`python-dev`).
   $ git submodule update --init
 
   # For the next two commands do `sudo pip install` if you get permission-denied errors
-  $ pip install -rrequirements.txt
+  $ pip install -requirements.txt
```

Comment:
This should actually be `-r requirements.txt`. `requirements.txt` is an argument to the `-r` flag. I _believe_ it should be working as it currently is, but isn't the most aesthetically pleasing.